hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1564,7 +1568,7 @@ protected void stopServiceThreads() {
   private void createProcedureExecutor() throws IOException {
     MasterProcedureEnv procEnv = new MasterProcedureEnv(this);
     procedureStore =
-      new WALProcedureStore(conf, new MasterProcedureEnv.WALStoreLeaseRecovery(this));
+      new RegionProcedureStore(this, new MasterProcedureEnv.FsUtilsLeaseRecovery(this));
```

Comment:
Use new RegionProcedureStore directly? Not a configable?

Reply:
Not needed for now. Can add a factory in the future if we have other implementations.